docs/tools/ntp.mdx

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
I'm not sure if this file should go in the "Software" or "Tools" section? We should confirm with Rhys when he is back. We can stick with Tools for now and it should be easy to move it later if needed.

Reply:
I put it with tools since it's peripherally related to the IP address stuff (which is also in tools), and `chrony` could be considered a tool. But I'm happy to relocate it if we want.

Reply:
Sounds good.